### PR TITLE
Create Orienteer reload page

### PR DESCRIPTION
**Pull request by WeaxMe**

### CODE REVIEW

File: orienteer-core/src/main/java/org/orienteer/core/web/ReloadOrienteerPage.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+      -moz-animation: progress 0s 1 forwards;
+      -ms-animation: progress 0s 1 forwards;
+      animation: progress 0s 1 forwards;
+    }
+
+    #pbaranim {
+      height: 15px;
+      width: 100%;
+      overflow: hidden;
+      background: url('/org/orienteer/core/web/reload/bar.gif') repeat-x;
```
Comment (PhantomYdn) [inline orienteer-core/src/main/java/org/orienteer/core/web/ReloadOrienteerPage.html]: Recommend to pack resource into the HTML through "data" protocol.

File: orienteer-core/src/main/java/org/orienteer/core/web/ReloadOrienteerPage.html [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        var id = setTimeout(function () {
+            const HTTP_UNAVAILABLE = 503;
+            var url = '/';
+            clearTimeout(id);
+            var request = function () {
+                var xhr = new XMLHttpRequest();
+                xhr.open('GET', url);
+                xhr.timeout = 5000;
+                xhr.addEventListener('loadend', function () {
+                    if (xhr.status !== HTTP_UNAVAILABLE) {
```
Comment (PhantomYdn) [inline orienteer-core/src/main/java/org/orienteer/core/web/ReloadOrienteerPage.html]: We might have here not only 503, but other error codes as well...
For now lets keep it, but probably some other changes might be needed. For example server might be not available at all.

File: orienteer-core/src/main/java/org/orienteer/core/web/ReloadOrienteerPage.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package org.orienteer.core.web;
```
Comment (PhantomYdn) [inline orienteer-core/src/main/java/org/orienteer/core/web/ReloadOrienteerPage.java]: Still need this page?

File: orienteer-core/src/main/resources/org/orienteer/core/web/reload/reload-page.html [hunk 4]
```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
```
Comment (PhantomYdn) [inline orienteer-core/src/main/resources/org/orienteer/core/web/reload/reload-page.html]: Code duplication? I saw the same below.

File: orienteer-core/src/main/java/org/orienteer/core/web/ReloadOrienteerPage.java (hunk 3 above)
Comment (WeaxMe) [inline orienteer-core/src/main/java/org/orienteer/core/web/ReloadOrienteerPage.java]: We need send WebPage to client when client click on "reload" button.

**Pull request merged**